_tests/integration/reactnative_test.go

```diff
@@ -447,22 +490,192 @@ warnings_with_recommendations:
+            receive your uploaded files.\n\n## Configure iOS part of the deploy workflow\n\nTo
+            generate IPA:\n\n1. Open the **Workflow** tab of your project on Bitrise.io\n1.
+            Click on **Code Signing** tab\n1. Find the **PROVISIONING PROFILE** section\n1.
+            Click or drop your file on the upload file field\n1. Find the **CODE SIGNING
+            IDENTITY** section\n1. Click or drop your file on the upload file field\n1.
+            Click on **Workflows** tab\n1. Select deploy workflow\n1. Select **Xcode Archive
+            & Export for iOS** step\n1. Open **Force Build Settings** input group\n1. Specify
+            codesign settings\nSet **Force code signing with Development Team**, **Force
+            code signing with Code Signing Identity**  \nand **Force code signing with Provisioning
+            Profile** inputs regarding to the uploaded codesigning files\n1. Specify manual
+            codesign style\nIf the codesigning files, are generated manually on the Apple
+            Developer Portal,  \nyou need to explicitly specify to use manual coedsign settings
+            \ \n(as ejected rn projects have xcode managed codesigning turned on).  \nTo
+            do so, add 'CODE_SIGN_STYLE=\"Manual\"' to 'Additional options for xcodebuild
+            call' input\n\n## To run this workflow\n\nIf you want to run this workflow manually:\n\n1.
+            Open the app's build list page\n2. Click on **[Start/Schedule a Build]** button\n3.
+            Select **deploy** in **Workflow** dropdown input\n4. Click **[Start Build]**
+            button\n\nOr if you need this workflow to be started by a GIT event:\n\n1. Click
+            on **Triggers** tab\n2. Setup your desired event (push/tag/pull) and select
+            **deploy** workflow\n3. Click on **[Done]** and then **[Save]** buttons\n\nThe
+            next change in your repository that matches any of your trigger map event will
+            start **deploy** workflow.\n"
+          steps:
+          - activate-ssh-key@%s:
+              run_if: '{{getenv "SSH_RSA_PRIVATE_KEY" | ne ""}}'
+          - git-clone@%s: {}
+          - npm@%s:
+              inputs:
+              - command: install
+          - install-missing-android-tools@%s:
+              inputs:
+              - gradlew_path: $PROJECT_LOCATION/gradlew
+          - android-build@%s:
+              inputs:
+              - project_location: $PROJECT_LOCATION
+          - xcode-archive@%s:
+              inputs:
+              - project_path: $BITRISE_PROJECT_PATH
+              - scheme: $BITRISE_SCHEME
+              - distribution_method: $BITRISE_DISTRIBUTION_METHOD
+              - configuration: Release
+              - automatic_code_signing: api-key
+          - deploy-to-bitrise-io@%s: {}
+        primary:
+          steps:
+          - activate-ssh-key@%s:
+              run_if: '{{getenv "SSH_RSA_PRIVATE_KEY" | ne ""}}'
```

Comment:
If you pull the latest master, this will be removed from the code generating the YAML, you should adjust these tests too